test/commands/render_test.rb

```diff
@@ -39,7 +39,7 @@ def test_render_for_toc
 
     assert_match /^## The table of contents/, markdown
     assert_match '<a href="#resource-app"', markdown
-    assert_match '- <a href="#link-POST-app-/apps">POST /apps', markdown
+    assert_match '- <a href="#app-create-app">POST /apps', markdown
     assert_match '<a name="link-POST-app-/apps"', markdown
```

Comment:
Wouldn't this `a name` need to change to match the other name for the anchors to function?